### PR TITLE
To do list structure PR

### PR DESCRIPTION
**Pull request by mertkantaroglu**
Hello,

For this project, I configured Webpack bundle and created main files with the JS functions. I also add basic styling to the file.

Looking forward to hearing from you.

Thanks,

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @mertkantaroglu,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly update your readme file with [the template](https://github.com/microverseinc/readme-template). Ensure you remove the logo and that all the table of content links are working properly.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,108 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank for Microverse to provide me learning materials, my peers for helping me out to move forward.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (roshan-bajgain) [inline README.md]: - [ ] You must delete or fill the FAQ section and every header from the [README template](https://github.com/microverseinc/readme-template) should be present. `Show your support` is missing please update it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,108 @@
...
+In order to run this project you need a Chrome, Safari or Mozilla browser
+
+### Setup <a name="setup"></a>
+
+Clone this repository to your desired folder: https://github.com/mertkantaroglu/To-Do-List-with-Webpack.git
+
+### Install <a name="install"></a>
+
+Install this project with Bash command
+
```
Comment (roshan-bajgain) [inline README.md]: - [x] I would suggest please write the proper comment like `git clone ...........` and installation part like npm install and to run that project use npm start. please write the proper command so anyone can run you project on browser.

**Review by roshan-bajgain — COMMENTED** [review]
Hi @mertkantaroglu, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no descriptive README file. please see below and update it

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+const toDoListContainer = document.querySelector('.to-do-list-container');
+
+const itemsListDisplay = () => {
+  itemsList.forEach((item) => {
+    toDoListContainer.innerHTML += `
+    <li class='to-do-item' value='index'>${item.description}</li>`;
+  });
+};
+
```
Comment (DammyShittu) [inline src/index.js]: - [x] Kindly ensure you style your list to match the wireframe.

  Your design| The wireframe
  :---:|:---:
  riow | kdkd

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @mertkantaroglu,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  <body>
+    <section class="main-container">
+      <div class="title">
+        <h3>Today's To Do</h3>
+        <i class="fa-solid fa-rotate fa-rotate-90"></i>
+      </div>
+      <div class="input-div">
+        <input type="text" id="input" placeholder="Add to your list.." />
+
+        <i class="fa-solid fa-arrow-turn-down fa-rotate-90"></i>
```
Comment (danmainah) [inline src/index.html:21]: - [x] @mertkantaroglu great work implementing the changes requested, however, the arrow button is slightly outside the  container , kindly ensure its inside the container to match the design given
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @mertkantaroglu , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 4 above)
Comment (mertkantaroglu) [reply src/index.html:21]: Hello Daniel,
Thanks for your feedback, but in my screen it shows in the container.
Screen Shot 2023-01-24 at 18 30 49

Can you please tell me how you looked at it? Then I can fix it accordingly.
Besides, is this really a big problem to postpone my approval? If you would say optional, I could fix it later on.

**Comment by mertkantaroglu** [comment]
Dear Reviewer,

In my screen design seems okay. You can see it below. Therefore, if there is any issues please let me know and I can fix it before you reject it.

Screen Shot 2023-01-24 at 18 30 49

Thank you.

**Comment by DammyShittu** [comment]
Hi @mertkantaroglu it is probably just a case of different browser. The chrome version of the app has the issue, but the firefox version of the app is just fine.

firefox browser | chrome browser
:---:|:---:
iew | odpe

**Review by DammyShittu — APPROVED** [review]
Hi @mertkantaroglu,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**